nussl/audio_signal.py

```diff
@@ -1157,7 +1220,7 @@ def get_magnitude_spectrogram_channel(self, n):
         # np.array helps with duck typing
         return utils._get_axis(np.array(self.magnitude_spectrogram_data), constants.STFT_CHAN_INDEX, n)
 
-    def to_mono(self, overwrite=False):
+    def to_mono(self, overwrite=False, remove_channels=True):
```

Comment:
What does this new parameter do again? I forget. We'll talk about if this is still needed, but you should at the very least document what you added. Make this a habit: write code --> write documentation!